README.md

```diff
@@ -468,4 +469,138 @@ class CalculateUserTransactionMetricsTestCase(TestCase):
 
 if __name__ == '__main__':
     main()
-```
\ No newline at end of file
+```
+
+### Creating beam manager
+Beam manager is an object that allows you to create dataflow pipelines using `create_dataflow_pipeline` method and 
+read/write data from bigquery and avro files using methods: `write_truncate_to_big_query`, `write_to_avro`, 
+`read_from_big_query` `read_from_avro`.
+Let's get through a few examples to illustrate each of those operations.
+
+Start with creation beam manager object. Parameters `project_id` and `dataset_name` defines dataset you want to work with.
+Parameter `internal_tables` specifies tables that are **inside** dataset specified by `project_id` and `dataset_name`.
+Parameter `external_tables` specifies tables that are **outside** dataset specified by `project_id` and `dataset_name`. 
+External tables have to be described by full table id, for example:
+
+```python
+external_tables = {
+    'transactions': 'dataset.id.transactions',
+    'some_external_table': 'dataset.id2.external_table'
+}
+```
+
+Parameter `runtime` is used to determine partition being processed.
+Parameter `dataflow_bucket` is  GCS bucket used for temporary and staging locations.
+Parameter `requirements_file_path` provides informations about dependencies of your dataflow.
+Parameter `region` is location of machine used in dataflow. By default is set to europe-west1.
```

Comment:
Is the location of the data center used to process your pipelines.

something like that :P

Reply:
Done